profiling: improve self-managed section (#3725)
elastic/observability-docs: docs/en/observability/profiling-self-managed.asciidoc

Both the collector and symbolizer need to authenticate to Elasticsearch to process profiling data.
For this, you need to create an API key for each application.

> Refer to https://www.elastic.co/guide/en/kibana/current/api-keys.html#create-api-key[Create an API key] to do it via Kibana.

Reviewer: ```suggestion
Refer to {kibana-ref}/current/api-keys.html#create-api-key[Create an API key] to create an API key using {kib}.
```

> You should select a "User API key", assigning the following permissions in the "Control security privileges" text area:

Reviewer: ```suggestion
Select a *User API key* and assign the following permissions  under *Control security privileges*:
```

# Configure the output to use when sending the data collected by pf-elastic-collector.

#-------------------------- Elasticsearch output --------------------------
output:
  elasticsearch:
    # Array of hosts to connect to.
    # Scheme and port can be left out and will be set to the default (`http` and `9200`).

>     # In case you specify and additional path, the scheme is required: `http://localhost:9200/path`.

Reviewer: ```suggestion
    # In case you specify an additional path, the scheme is required: `http://localhost:9200/path`.
```
